experimental/GModule/src/Cohomology.jl

```diff
@@ -547,7 +547,7 @@ export is_stem_extension, is_central
 
 _rank(M::FinGenAbGroup) = torsion_free_rank(M)
 _rank(M) = rank(M)
-_rank(M::AbstractAlgebra.FPModule{<:FieldElem}) = dim(M)
+_rank(M::AbstractAlgebra.FPModule{<:FieldElem}) = vector_space_dim(M)
```

Comment:
Don't `rank` and `vector_space_dim` return the same thing here? If so, I'd find this more sensible:
```suggestion
_rank(M::AbstractAlgebra.FPModule{<:FieldElem}) = rank(M)
```

Reply:
Oh no wait, I take that back, I was confusing `FPModule` with `FreeModule` in my brain. Sorry for the noise